API.md

```diff
@@ -22,31 +22,31 @@ Implements methods to deal with diff output.
 **Kind**: global class  
 
 * [AsyncAPIDiff](#AsyncAPIDiff)
-    * [.breaking()](#AsyncAPIDiff+breaking) ⇒ <code>Array.&lt;DiffOutputItem&gt;</code>
-    * [.nonBreaking()](#AsyncAPIDiff+nonBreaking) ⇒ <code>Array.&lt;DiffOutputItem&gt;</code>
-    * [.unclassified()](#AsyncAPIDiff+unclassified) ⇒ <code>Array.&lt;DiffOutputItem&gt;</code>
-    * [.getOutput()](#AsyncAPIDiff+getOutput) ⇒ <code>Output</code>
+    * [.breaking()](#AsyncAPIDiff+breaking) ⇒ <code>Array.&lt;DiffOutputItem&gt;</code> \| <code>string</code>
```

Comment:
You don't need to push that changes, because it will be done by our bot after merge. Please revert that file :)